renovate.json

```diff
@@ -2,6 +2,7 @@
 	"extends": [ "config:base" ],
 	"lockFileMaintenance": { "enabled": true },
 	"ignoreDeps": [ "husky" ],
+	"prConcurrentLimit": 5,
```

Comment:
Initially, it might potentially take a while to get through all updates with the concurrent limit in place. What are your thoughts on maybe adding [prPriority](https://docs.renovatebot.com/configuration-options/#prpriority) to the package rules so the most important updates go first?